pingora-limits/src/estimator.rs

```diff
@@ -30,33 +30,28 @@ pub struct Estimator {
 impl Estimator {
     /// Create a new `Estimator` with the given amount of hashes and columns (slots).
     pub fn new(hashes: usize, slots: usize) -> Self {
-        let mut estimator = Vec::with_capacity(hashes);
-        for _ in 0..hashes {
-            let mut slot = Vec::with_capacity(slots);
-            for _ in 0..slots {
-                slot.push(AtomicIsize::new(0));
-            }
-            estimator.push((slot.into_boxed_slice(), RandomState::new()));
-        }
-
-        Estimator {
-            estimator: estimator.into_boxed_slice(),
+        Self {
+            estimator: (0..hashes)
+                .map(|_| (0..slots).map(|_| AtomicIsize::new(0)).collect::<Vec<_>>())
```

Comment:
Could also do something like this 

```suggestion
                .map(|_| vec![AtomicIsize::new(0); slots])
```

Reply:
I had tried this, doesn't work because `AtomicIsize` doesn't implement Clone

Reply:
Ah! Sorry for the invalid suggestion.